examples/zookeeper/config-and-run.sh

```diff
@@ -1,5 +1,13 @@
 #!/bin/bash
 
+# Make sure conf folder exists
+if [[ ! -d /opt/zookeeper/conf ]]; then
+  mkdir -p /opt/zookeeper/conf
+fi
+
+# Copy config files if conf folder is empty
+cp --no-clobber /opt/zookeeper/conf.template/* /opt/zookeeper/conf/
```

Comment:
You're right. The script should copy the files over only when the directory is completely empty (i.e., the first time the container is run with volumes attached).